### PR TITLE
Add missing Obsoletes: tfm-ror52 line

### PR DESCRIPTION
**Pull request by ekohl**

### CODE REVIEW

**Comment by ekohl** [comment]
This is part of the base install which suggests https://github.com/theforeman/forklift/blob/8dea923b2373d745486077c93702cf00177a6115/bats/fb-verify-packages.bats#L14 is not doing anything

File: packages/foreman/rubygem-thread_safe/rubygem-thread_safe.spec [hunk 1]
```diff
@@ -24,6 +24,8 @@ BuildArch: noarch
 Provides: %{?scl_prefix}rubygem(%{gem_name}) = %{version}
 # end specfile generated dependencies
 
+Obsoletes: tfm-ror52-rubygem-%{gem_name} <= 0.3.6
```
Comment (evgeni) [inline packages/foreman/rubygem-thread_safe/rubygem-thread_safe.spec:27]: I think it should at least obsolete 0.3.6-4?

File: packages/foreman/rubygem-thor/rubygem-thor.spec [hunk 2]
```diff
@@ -24,6 +24,8 @@ BuildArch: noarch
 Provides: %{?scl_prefix}rubygem(%{gem_name}) = %{version}
 # end specfile generated dependencies
 
+Obsoletes: tfm-ror52-rubygem-%{gem_name} <= 1.0.1
```
Comment (evgeni) [inline packages/foreman/rubygem-thor/rubygem-thor.spec:27]: not 1.0.1-1?

Comment (evgeni) [reply packages/foreman/rubygem-thor/rubygem-thor.spec:27]: or rather "< %{version}-%{release}"?

File: packages/foreman/rubygem-thread_safe/rubygem-thread_safe.spec (hunk 1 above)
Comment (ekohl) [reply packages/foreman/rubygem-thread_safe/rubygem-thread_safe.spec:27]: I copied this from all other obsoletes. Perhaps they all need to be modified?

File: packages/foreman/rubygem-thor/rubygem-thor.spec (hunk 2 above)
Comment (ekohl) [reply packages/foreman/rubygem-thor/rubygem-thor.spec:27]: I considered the same

**Comment by ekohl** [comment]
The other issue I found on my server was:
```console
# rpm -qv tfm-{ror52-,}rubygem-multi_json
tfm-ror52-rubygem-multi_json-1.13.1-1.el7.noarch
tfm-rubygem-multi_json-1.12.2-5.el7.noarch
```

Because it's a downgrade, the ror52 package isn't obsoleted.

**Comment by evgeni** [comment]
It shouldn't be an downgrade… ;)

**Comment by evgeni** [comment]
continuing in https://github.com/theforeman/foreman-packaging/pull/5286

**Pull request closed without merging**